src/migrations/20240830102144-onboarding-events.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+            CREATE TABLE onboarding_events (
+               event VARCHAR(50) NOT NULL,
+               diff INTEGER NOT NULL,
```

Comment:
In seconds, rounded, can not be empty

Reply:
is it diff from project created or first user created?